src/components/NavBar.jsx

```diff
@@ -0,0 +1,33 @@
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav';
+import Form from 'react-bootstrap/Form';
+import FormControl from 'react-bootstrap/FormControl';
+import Button from 'react-bootstrap/Button';
+
+const NavBar = () => {
+  return (
+    <Navbar collapseOnSelect expand="false" bg="success" variant="dark">
+      <Form className="d-flex m-2" style={{ maxWidth: '70%' }}>
+        <FormControl
+          type="search"
+          placeholder="Search"
+          className="me-2"
+          aria-label="Search"
+        />
+        <Button variant="danger">Search</Button>
+      </Form>
```

Comment:
For this you can create a component that handles everything related to the form. As in the future you will need to add logic to search videos